src/modules/material-requests/components/MaterialRequestDetailBlade.tsx

```diff
@@ -0,0 +1,152 @@
+					</p>
+				</div>
+			</div>
+			<a
+				href={`/${currentMaterialRequestDetails?.maintainerSlug}/${currentMaterialRequestDetails?.objectSchemaIdentifier}`}
+				className="p-account-my-material-requests__material-link"
+			>
+				<div className="p-account-my-material-requests__material">
+					<p className="p-account-my-material-requests__material-label">
+						<Icon
+							className="p-account-my-material-requests__material-label-icon"
+							// TODO: make icon dynamic when objectType is returned
+							name={IconNamesLight.Video}
+						/>
+						{currentMaterialRequestDetails?.objectSchemaName}
+					</p>
+					<p className="p-account-my-material-requests__material-id">
+						{currentMaterialRequestDetails?.objectSchemaIdentifier}
+					</p>
+				</div>
+			</a>
+			<div className="p-account-my-material-requests__content">
+				{currentMaterialRequestDetails?.requesterFullName && (
+					<>
+						<p className="p-account-my-material-requests__content-label">
+							{tText('modules/cp/const/material-requests___naam')}
+						</p>
+						<p className="p-account-my-material-requests__content-value">
+							{currentMaterialRequestDetails.requesterFullName}
+						</p>
+					</>
+				)}
+				{currentMaterialRequestDetails?.requesterMail && (
+					<>
+						<p className="p-account-my-material-requests__content-label">
+							{tText('modules/cp/const/material-requests___emailadres')}
+						</p>
+						<p className="p-account-my-material-requests__content-value">
+							{currentMaterialRequestDetails.requesterMail}
+						</p>
+					</>
+				)}
+				{currentMaterialRequestDetails?.organisation && (
+					<>
+						<p className="p-account-my-material-requests__content-label">
+							{tText('modules/cp/const/material-requests___organisatie')}
+						</p>
+						<p className="p-account-my-material-requests__content-value">
+							{currentMaterialRequestDetails.organisation}
+						</p>
+					</>
+				)}
+				{currentMaterialRequestDetails?.reason && (
+					<>
+						<p className="p-account-my-material-requests__content-label">
+							{tText('modules/cp/const/material-requests___reden')}
+						</p>
+						<p className="p-account-my-material-requests__content-value">
+							{currentMaterialRequestDetails.reason}
+						</p>
+					</>
+				)}
+				{currentMaterialRequestDetails?.createdAt && (
+					<>
+						<p className="p-account-my-material-requests__content-label">
+							{tText('modules/cp/const/material-requests___datum')}
+						</p>
+						<p className="p-account-my-material-requests__content-value">
+							{formatMediumDate(new Date(currentMaterialRequestDetails.createdAt))}
+						</p>
+					</>
+				)}
+				{currentMaterialRequestDetails?.type && (
+					<>
+						<p className="p-account-my-material-requests__content-label">
+							{tText('modules/cp/const/material-requests___type-aanvraag')}
+						</p>
+						<p className="p-account-my-material-requests__content-value">
+							{currentMaterialRequestDetails.type}
```

Comment:
You should use the translation key for this. There's already a record in the codebase which you can reuse:
```
{MATERIAL_REQUEST_TRANSLATIONS_BY_TYPE[currentMaterialRequestDetails.type]}
``